clientconn_test.go

```diff
@@ -335,7 +335,7 @@ func (s) TestCloseConnectionWhenServerPrefaceNotReceived(t *testing.T) {
 func (s) TestBackoffWhenNoServerPrefaceReceived(t *testing.T) {
 	lis, err := net.Listen("tcp", "localhost:0")
 	if err != nil {
-		t.Fatalf("Error while listening. Err: %v", err)
+		t.Fatalf("net.Listen() = %v", err)
```

Comment:
Sorry to be harsh, but I just don't see value in doing this style of change.  Nothing was improved.  In fact, I find this even less informative.  It tells me something happened, but not what was expected to happen.  I.e. I can't tell from this message whether the problem was that the error was expected to be a different error or no error at all.

IMO ideal would be more like `Unexpected error from net.Listen("tcp", "localhost:0"): %v`.